frontend/review/src/components/GraphBox.js

```diff
@@ -0,0 +1,501 @@
+          },
+        },
+
+        scaleLabel: {
+          display: true,
+          labelString: "Percent Through Add/Drop Period",
+        },
+      },
+    ],
+    yAxes: [
+      {
+        display: true,
+        ticks: {
+          autoSkip: true,
+          maxTicksLimit: 10,
+          maxRotation: 0,
+          minRotation: 0,
+          min: 0,
+          max: 1,
+        },
+      },
+    ],
+  },
+};
+
+const percentSectionChartOptions = {
+  tooltips: {
+    mode: "index",
+    intersect: false,
+    backgroundColor: "#E9E9E9",
+    bodyFontColor: "#000000",
+    titleFontColor: "#000000",
+    cornerRadius: 3,
+    bodyFontSize: 12,
+    custom: function(tooltip) {
+      if (!tooltip) return;
+      // disable displaying the color box;
+      tooltip.displayColors = false;
+    },
+    callbacks: {
+      title: (toolTipItem, data) => {
+        return (
+          data["datasets"][0]["data"][toolTipItem[0]["index"]].x +
+          "% Through Add/Drop"
+        );
+      },
+      beforeBody: (toolTipItem, data) => {
+        let bodyText = "";
+        bodyText +=
+          "Projected Date: " +
+          calcApproxDate(
+            addDropDate.start,
+            addDropDate.end,
+            data["datasets"][0]["data"][toolTipItem[0]["index"]].x / 100
+          ) +
+          "\n";
+        bodyText +=
+          "% of Sections Open: " +
+          data["datasets"][0]["data"][toolTipItem[0]["index"]].y +
+          "%";
+        return bodyText;
+      },
+      label: () => {
+        return;
+      },
+    },
+  },
+  hover: {
+    mode: "nearest",
+    intersect: true,
+  },
+  elements: {
+    point: {
+      radius: 0,
+    },
+  },
+  legend: {
+    display: true,
+    position: "bottom",
+    align: "start",
+  },
+  scales: {
+    autoSkip: true,
+    xAxes: [
+      {
+        display: true,
+        ticks: {
+          autoSkip: true,
+          beginAtZero: true,
+          maxTicksLimit: 6,
+          stepSize: 20,
+          maxRotation: 0,
+          minRotation: 0,
+          max: 100,
+          callback: (value) => {
+            return value + "%";
+          },
+        },
+
+        scaleLabel: {
+          display: true,
+          labelString: "Percent Through Add/Drop Period",
+        },
+      },
+    ],
+    yAxes: [
+      {
+        display: true,
+        ticks: {
+          autoSkip: true,
+          maxTicksLimit: 5,
+          stepSize: 25,
+          maxRotation: 0,
+          minRotation: 0,
+          min: 0,
+          max: 100,
+          callback: (value) => {
+            return value + "%";
+          },
+        },
+      },
+    ],
+  },
+};
+
+const translateSemester = (semester) => {
+  const year = semester.substring(0, 4);
+  const season = semester.substring(4);
+  if (season === "A") {
+    return "Spring " + year;
+  } else if (season === "B") {
+    return "Summer " + year;
+  } else if (season === "C") {
+    return "Fall " + year;
+  }
+  return year;
+};
```

Comment:
check in the utils file - I think there's already a function that does this, or something very close you can take advantage of.